applications/StructuralMechanicsApplication/tests/test_dynamic_schemes.py

```diff
@@ -20,21 +20,20 @@ def _base_spring_test_pseudo_static_scheme(self, current_model, scheme_name = "p
         node.AddDof(KratosMultiphysics.DISPLACEMENT_Y)
         node.AddDof(KratosMultiphysics.DISPLACEMENT_Z)
 
-        #add bcs and initial values
+        # Add bcs and initial values
```

Comment:
question: is there any check that verifies that imposition of VElOCITY and/or ACCELERATION works correctly? (and that should be with a extremely tight tolerance)

Reply:
The cpp test for test_schemes does that with three predict tests (one for each derivative level)